pynucastro/networks/tests/_starkiller_cxx_reference/table_rates.H

```diff
@@ -67,10 +66,10 @@ namespace rate_tables
 }
 
 template <typename R, typename T, typename D>
-void init_tab_info(const table_t tf, R& rhoy, T& temp, D& data) {
+void init_tab_info(const table_t tf, const std::string file, R& rhoy, T& temp, D& data) {
```

Comment:
doesn't much matter, but perhaps we should do `const std::string& file` to save the copy

Reply:
That makes sense, I'll add that change